Automattic/jetpack
# Fix/my jetpack async activate

## Proposed changes:

* This diff removes some empty href attributes for certain button states of My Jetpack cards that were causing the the page to get reloaded when the card action was clicked. This was happening for "Activate" and "Install" options

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Check out this branch locally or on a test site with the beta plugin
* Install the Boost plugin, but don't activate it yet
* Go to My Jetpack
* The boost card should show in "My products" with an "Activate" CTA
![Screenshot 2024-08-12 at 4 01 14 PM](https://github.com/user-attachments/assets/1c2334ae-1ade-4a95-8d44-a885e48369be)
* Click "Activate"
* The card should activate the plugin asynchronously without reloading the page. The card status indicator should switch to active
![Screenshot 2024-08-12 at 4 09 21 PM](https://github.com/user-attachments/assets/7d0ee34d-a5df-4274-9604-4a4a27a58c7b)
